plugins/MantisGraph/lang/strings_english.txt

```diff
@@ -88,3 +88,6 @@ $s_plugin_MantisGraph_graph_topdev = 'Top Developers by Fixed Issues';
 $s_plugin_MantisGraph_graph_opendev = 'Developers by Open Issues';
 $s_plugin_MantisGraph_graph_topreporter_fixed = 'Top Reporters by Fixed Issues';
 $s_plugin_MantisGraph_other_categories = 'other categories (%d)';
+
+$s_plugin_MantisGraph_other_projects = 'other projecst (%d)';
+$s_plugin_MantisGraph_graph_imp_project_title = 'By Projects Graphs';
```

Comment:
Not sure why you're tagging these 2 strings at the end, it would be more consistent to put them together with related strings. Also the `imp` prefix is probably the result of a copy/paste, not needed IMO (TBH, I'm not even what its meaning is/was).